cypress/e2e/crypto/crypto.spec.ts

```diff
@@ -415,10 +415,12 @@ describe("Cryptography", function () {
                 .then(() => bobSecondDevice.logout(true))
                 .then(() => cy.log(`Bob logged out second device`));
 
+            // some debate over whether this should have a red or a grey shield. Legacy crypto shows a grey shield,
+            // Rust crypto a red one.
             cy.get(".mx_EventTile_last")
                 .should("contain", "test encrypted from unverified")
                 .find(".mx_EventTile_e2eIcon")
-                .should("have.class", "mx_EventTile_e2eIcon_normal")
+                //.should("have.class", "mx_EventTile_e2eIcon_normal")
```

Comment:
Should this be removed rather than comment out?

Reply:
hrm, maybe. I'm trying to highlight the fact that we deliberately aren't testing the class (and also, making it easier to put back once we make our mind up what we're doing).